src/utils/IdealGraphVisualizer/Graal/src/main/java/com/sun/hotspot/igv/graal/filters/GraalEdgeColorFilter.java

```diff
@@ -57,11 +57,8 @@ public void apply(Diagram d) {
         List<Figure> figures = d.getFigures();
         for (Figure f : figures) {
             for (InputSlot is : f.getInputSlots()) {
-                for (Connection c : is.getConnections()) {
-                    String type = c.getType();
-                    if (type == "Association" && "EndNode".equals(c.getOutputSlot().getFigure().getProperties().get("class"))) {
-                        type = "Successor";
-                    }
+                for (FigureConnection c : is.getConnections()) {
+                    String type = null;
 
                     if (type != null) {
```

Comment:
I removed the method `Connection::getType()` and the code that depends on it in the Graal filters (the only place where this method was used). The Graal filters are not used and should just get removed, since HotSpot IGV does not support Graal anymore (a specific IGV variant is available for Graal, see https://www.graalvm.org/advanced-tools/). I will file a separate cleanup RFE for that.

Reply:
I just filed [JDK-8283612](https://bugs.openjdk.java.net/browse/JDK-8283612) for removing the Graal module.